### ISSUE TITLE
Error messages when running Vim with Cygwin.

### ISSUE THREAD

**Issue by aserebryakov** [aserebryakov/you-debug-it#1]
Errors when running the vim with Cygwin.

```
Error detected while processing /home/user/.vim/bundle/you-debug-it/plugin/you-debug-it.vim:
line   14:
E492: Not an editor command: ^M
line   16:
E492: Not an editor command: ^M
line   17:
E492: Not an editor command: ^M
line   19:
E488: Trailing characters
line   23:
E492: Not an editor command: ^M
line   24:
E492: Not an editor command: ^M
line   26:
E488: Trailing characters
line   30:
E492: Not an editor command: ^M
line   31:
E492: Not an editor command: ^M
line   33:
E488: Trailing characters
line   36:
E492: Not an editor command: ^M
line   37:
E492: Not an editor command: ^M
line   39:
E15: Invalid expression: !exists('g:youdebugit_plugin')^M
line   48:
E171: Missing :endif
Press ENTER or type command to continue

```

**Comment by aserebryakov** [comment]
The issue caused by DOS line endings in plugin sources.

**Comment by aserebryakov** [comment]
Resolved by d0f3f3a074967777273bfb854a01b75f1167e81d.
